packages/aws-cdk-lib/core/lib/tag-aspect.ts

```diff
@@ -159,7 +159,7 @@ export class Tags {
    * add tags to the node of a construct and all its the taggable children
    */
   public add(key: string, value: string, props: TagProps = {}) {
-    Aspects.of(this.scope).add(new Tag(key, value, props)), { priority: AspectPriority.MUTATING };
+    Aspects.of(this.scope).add(new Tag(key, value, props), { priority: AspectPriority.MUTATING });
```

Comment:
Why not break this into a few more statements to improve readability and reduce the likelihood of an error like this in the future.  Maybe something like:
```suggestion
    const newTag = new Tag(key, value, props);
    const priorityOptions = { priority: AspectPriority.MUTATING };
    Aspects.of(this.scope).add(newTag, priorityOptions);
```